lib/spack/spack/package.py

```diff
@@ -52,6 +52,7 @@
 import spack.store
 import spack.url
 import spack.util.environment
+import spack.util.hash
```

Comment:
Why is this imported here if the rest of the changes in this file do not use that import? Is it getting magically invoked somehow?

Reply:
I am very confused by this -- it seems like flake8 is overlooking it.

Reply:
I just removed this.